### DISCUSSION TITLE
2.1.9 - fix vpn service, wireguard

### DISCUSSION THREAD

**Discussion by znetworkx** [keenetic-dev/bypass_keenetic#30] [Announcements]
- *Исправлены ошибки в скриптах для VPN*
- *Протестирована и настроена работа с Wireguard*
- *Исправлены мелкие ошибки*
- *В бот добавлена информация `где брать ключи`*
- *Обновлено меню*
- *Добавлены реквизиты для доната*

**Full Changelog**: https://github.com/ziwork/bypass_keenetic/compare/2.1.8...2.1.9

<hr /><em>This discussion was created from the release <a href='https://github.com/ziwork/bypass_keenetic/releases/tag/2.1.9'>2.1.9 - fix vpn service, wireguard</a>.</em>

**Comment by ikocs** [comment]
Подскажите, а как настроить Wireguard?
В боте доп.меню в мостах не вижу

**Reply by znetworkx** [reply]
Добрый вечер, @ikocs.

WireGuard настраивается на самом роутере, инструкция тут: 
https://help.keenetic.com/hc/ru/articles/360010592379-WireGuard-VPN

Далее, после настройки и подключения, в боте (Меню->Списки) появится меню для добавления в него адресов.

**Reply by ikocs** [reply]
а, все, увидел. У меня на роутере уже настроен, думал в боте надо отдельно.

Два вопроса тогда ещё:
1. У меня сейчас настроен `shadowsocks` и в нём есть список обхода. Если у меня будет такой же список для `WG`, то какое туннель будет в приоритете выбираться?
2. Если поведения в пункте `1.` непредсказуемое, то самый простой способ очистить список обхода для `shadowsocks`, чтобы стабильно использовать `WG`?

**Reply by znetworkx** [reply]
1. Трафик через WG будет приоритетнее.
2. Самый простой способ очистки без потери данных:
Меню - Список - SS - Показать список - копируем, - Назад.
Меню - Список - SS - Удалить из списка - вставляем то, что скопировали, отправляем боту.
Список будет пуст после этого.

**Reply by ikocs** [reply]
Всё заработало, спасибо. WG всё-таки пошустрее у меня работает.

**Comment by ikocs** [comment]
Я ещё с одним вопросом. 
Сегодня роутер начал заворачивать вообще весь трафик через VPN. Не помогло ни отключение DNS override через бота, ни удаление каких-то сайтов из листа обхода. Не сталкивались с таким?

Пока временно вынул флешку

**Reply by ikocs** [reply]
Так, судя по всему, все ломало добавление в обход news.google.com
